scripts/generate_checklist.py

```
@@ -0,0 +1,90 @@
+        raise RuntimeError(p.stderr or p.stdout)
+    return p.stdout.strip()
+
+
+def list_issues(milestone: str, state: str = "open"):
+    # gh issue list supports --milestone
+    return sh_json(["gh", "issue", "list", "--state", state, "--limit", "500", "--milestone", milestone,
+                    "--json", "number,title,state,url,labels,milestone"]) or []
+
+
+def is_epic(issue):
+    return any(l.get("name") == "epic" for l in (issue.get("labels") or []))
```

Ruff (0.14.2)

Ambiguous variable name: `l`

(E741)